NEWS

```diff
@@ -6,6 +6,10 @@ NEWS - user visible changes				-*- outline -*-
 
 * New GnuCOBOL features
 
+** the leading space for all internal directives is removed in the lexer.
+   Source previously processed may need to be ajusted to be processed by
```

Comment:
typo: "adjusted"; the whole block needs to be moved down to "Changes that potentially effect recompilation of existing programs:" and can skip the "by gc 3.3" part; as this is only about **pre**processed files  (please adjust the text) that whole block is only important for the NEWS file if any GC version created leading spaces before the internal directives - is this the case? (To be sure it should be enough to test 1 file with some directives in GC 2.2, 3.1 and 3.2, I _think_ there weren't such cases)

Reply:
I compiled some tests from the testsuite with the `-E` option. The directives appear but no leading space for 2.2, 3.1 and 3.2. Should I remove the whole block from the NEWS file?

Reply:
In this case I'd suggest to do that - we still have Changelogs for these kind of things :-)